GSOC18/GeometryUtilities-master/interpolation/test_coef.py

```diff
@@ -0,0 +1,138 @@
+##########################IMPORTS########################
+#For timing script
+import datetime
+#For file IO/data Handling
+import os
+import cPickle as pickle
+#import pandas as pd
+#Linear Algebra library
+import numpy as np
+from scipy.spatial import  cKDTree
+#Plotting Imports and configuration
+import matplotlib.pyplot as plt
+from shapely.geometry import LineString,Polygon
+from descartes.patch import PolygonPatch
+#Geometry File imports
+from geometry.cmssw import read_geometry
+#Importing custom classes and function
+from sq_Cells import sq_Cells
+from scipy import misc
+
+# Parameters ... to fix later
+#input_default_file = '/data_CMS/cms/grasseau/HAhRD/test_triggergeom.root'
+input_default_file='geometry_data/test_triggergeom.root'
+coef_filename='sq_cells_data/coef_dict_layer_1_len_0.7.pkl'
+#This need to be manually entered now from the screen output in main script
+resolution = (442,442)

```

Comment:
This resolution here needs to be set by hand(manually) for now by looking at the output of the main.py script I print the resolution after calculation from the exp_edge_length.